internal/proto/stdout_stderr_writer.go

```diff
@@ -0,0 +1,133 @@
+package proto
+
+import (
+	"bytes"
+	"io"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"os"
+)
+
+type StdoutStderrWriter interface {
+	GetStdoutWriter() io.Writer
```

Comment:
in idiomatic Go, we usually don't include "Get" or the type name in the method name
https://golang.org/doc/effective_go.html#Getters
so this would really be called `Stdout() io.Writer`